templates/vue/src/components/Lightbox/media/common/CompletedActivityMedia/index.vue

```diff
@@ -77,7 +77,8 @@ export default {
 
 <style lang="scss" scoped>
 .completed-activity-media {
-  background: #262626;
+  background: #eeeeee;
+  color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
yeah, it looks like only the background is necessary... everything else is inherited.